packages/amazon-sumerian-hosts-babylon/package.json

```diff
@@ -24,6 +24,7 @@
     "aws-sdk": "^2.1094.0"
   },
   "peerDependencies": {
-    "babylonjs": "=4.2.1"
+    "@babylonjs/core": "=4.2.1",
```

Comment:
do we want `=`? Should we use `^` so naturally test with newer versions?
Also..babylon is on version 5. we should be using that now right? are we tracking this?

Reply:
`4.2.1` is the latest/last version of 4.x before 5.x; `4.2.0` has a bug with .glb loaders, thus my decision not to use `<=`

BabylonJS 5.x won't be officially released until May -- and introduces some breaking changes, so we'd need to update our library to work with it. (I spent some time testing our repo with 5.2.0) The BabylonJS Editor targets 4.2.1, thus my "locking" it to this change

Reply:
ah ok. should we do `^4.2.1` then?

and didn't realize it wasn't out yet. 5 looks to be the latest on NPM so maybe I misunderstand what 'release' means :)

Reply:
Haha, yeah - I mispoke a bit - [the forums banner](https://forum.babylonjs.com/) just says they won't be acknowledging its release until May.